cuba.yml

```diff
@@ -545,3 +545,23 @@ CUBA_KEYS:
     definition: Classification of a fluid flow type
     shape: [1]
     type: integer
+
+  VECTOR:
+    definition: A vector
+    shape: [1]
+    type: double
+
+  POINT:
+    definition: A point in a 3D space system
+    shape: [3]
+    type: double
```

Comment:
> CUBA.POSITION is defined in cuba.yml, is that what you want?
a position is defined by a point. An entity can have a position, and this position is defined by a point in space. For example, a group of atoms may have a position that is defined as the centre of mass: group --> centre of mass --> position --> point.